drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/strategies/DMNDataManagementStrategy.java

```diff
@@ -93,19 +90,19 @@ private void showErrorsAndCleanupState(FactModelTuple factModelTuple) {
         if (factModelTuple.getTopLevelCollectionError().size() > 0) {
```

Comment:
Please remove this block (now they are supported)

Reply:
Done